mlvm/README.md

```diff
@@ -0,0 +1,153 @@
+# Machine Learning VM
+
+This initialization action installs a set of packages designed to get you up and running with many commonly-used machine learning packages on a
+[Dataproc](https://cloud.google.com/dataproc) cluster. Currently, this initialization action is supported on Dataproc [Ubuntu images] (https://cloud.google.com/dataproc/docs/concepts/versioning/dataproc-versions) 1.5 or higher. It is recommended to use our 2.0 image track for the best performance and greatest access to features. 
+
+Features of this configuration include:
+
+* Python packages such as  TensorFlow, PyTorch, MxNet, Scikit-learn and Keras
+* R packages including XGBoost, Caret, randomForest, sparklyr
+* Full list of Python and R packages below.
+* Spark-BigQuery Connector
+* RAPIDS on Spark and GPU support
+* Jupyter and Anaconda via [Optional Components](https://cloud.google.com/dataproc/docs/concepts/components/overview)
+* [Component Gateway](https://cloud.google.com/dataproc/docs/concepts/accessing/dataproc-gateways)
+
+## Using this initialization action
+
+**:warning: NOTICE:** See [best practices](/README.md#how-initialization-actions-are-used) of using initialization actions in production.
+
+You can use this initialization action to create a new Dataproc cluster with a set of preconfigured machine learning packages. You'll need to supply the following metadata flags:
+
+```
+--metadata spark-bigquery-connector-version=0.17.0
+```
+Feel free to use a different connector version. 
+
+If you wish to have GPU and/or RAPIDS support, include the following.
+```
+--metadata gpu-driver-provider=NVIDIA
+--metadata include-gpus=true
+--metadata rapids-runtime=<SPARK|DASK>
+
+# For RAPIDS on Dataproc 1.5
+--metadata cuda-version=10.1
+```
+
+You can also include your own Python libraries with the metadata flag `PYTHON_PACKAGES`.
+```
+--metadata PYTHON_PACKAGES=package1==ver1,package2==ver2
+```
+
+
+1.  Use the `gcloud` command to create a new cluster with this initialization action. The command shown below includes a curated list of packages that will be installed on the cluster:
+
+    ```bash
+    REGION=<region>
+    CLUSTER_NAME=<cluster_name>
+    gcloud dataproc clusters create ${CLUSTER_NAME} \
+        --region ${REGION} \
+        --master-machine-type n1-standard-16 \
+        --worker-machine-type n1-highmem-32 \
+        --worker-accelerator type=nvidia-tesla-t4,count=2 \
+        --image-version preview-ubuntu \
+        --metadata spark-bigquery-connector-version=0.17.0 \
```

Comment:
Update to 0.17.1

Reply:
I think I made an error below, I don't believe 0.17.1 is released yet. I'm also going to move this functionality into the init. action as this is a repetitive step.